lib/packwerk/parse_run.rb

```diff
@@ -57,6 +58,20 @@ def update_todo
       Cli::Result.new(message: message, status: offense_collection.errors.empty?)
     end
 
+    sig { returns(Cli::Result) }
+    def count_offenses
+      run_context = RunContext.from_configuration(@configuration)
+      offense_collection = find_offenses(run_context)
+
+      message = CSV.generate do |csv|
+        offense_collection.package_todos.each do |package, package_todo|
+          csv << [package.name, package_todo.count]
+        end
+      end
+
+      Cli::Result.new(message: message, status: true)
```

Comment:
I think we talked about this a little, but do you think we could achieve the same thing with an offense formatter? Also do you think CSV is a readable default for this command? Maybe we should print them out by default, and then offer CSV as a flag.

Reply:
Somehow, GitHub managed to get this comment to point to something in the PR, but please note that the method is not `#count_offenses` anymore, but `#show_offenses`. It uses an offense formatter.